releasenotes/notes/replcmds-3a95564a5f58197b.yaml

```diff
@@ -0,0 +1,10 @@
+---
+deprecations:
+  - |
+    Begin deprecating "Agent start".  It is being replaced by "run".  Command
```

Comment:
nit: lowercase agent in "agent start" :)

Also, let's mention that it's the agent start _command_ in the 1st sentence